rest-api/src/test/java/gov/cms/qpp/conversion/api/security/JwtAuthorizationFilterTest.java

```diff
@@ -27,6 +29,7 @@
 //Using jUnit 4 for power mock
 @RunWith(PowerMockRunner.class)
 @PrepareForTest({SecurityContextHolder.class})
+@PowerMockIgnore("javax.crypto.*")
```

Comment:
Does this have to be a * dependency? otherwise I'm fine with these changes.

Reply:
Let me see if I can be more precise.

Reply:
I cannot seem to be more precise.  It continues to throw the same error when I try.

Reply:
K. That is fine.